src/reducers/angler_reducers.js

```diff
@@ -0,0 +1,16 @@
+import {
+    FETCH_ANGLERS_SUCCESS
+} from '../actions/types';
+
+export default (state=[], action) => {
+    switch(action.type) {
+        case FETCH_ANGLERS_SUCCESS:
+            const unsortedFish = action.payload.data.map(fish => {
```

Comment:
you will want to rethink this. part of the redux flow is immutable data, so when you add a property to fish, eg `fish.weight`, you violate that principle.